src/compiler/prioritization.md

```diff
@@ -1,3 +1,230 @@
 # Prioritization
+It is important that the compiler team can quickly identify priority issues, hence the establishment
+of a prioritization process, described below.
 
-This section documents the processes of the prioritization WG.
+## General Process
+1. Ascertain the current status of the issue
+1. Try progressing the issue if possible (e.g. request updates from the issue author/reviewer)
+1. Is there an MCVE for the issue already?
+1. Check if it's a regression and label it accordingly (`regression-*` labels)
+1. Figure out the area the issue belongs and label it accordingly (`A-*` labels)
+1. [Ping notify groups](https://rustc-dev-guide.rust-lang.org/notification-groups/about.html) or
+   relevant teams
+1. Assign if possible
+
+# Priority Levels
+As the compiler team's resources are limited, the primary goal of prioritization is to identify the
+most relevant issues to work on, so that the compiler team can focus on what matters the most.
+
+Issues relevant to prioritization are bugs and feature requests that are nominated for
+prioritization, by adding the `I-prioritize` label as described below.
+
+## Labels
+Labeling an issue as `I-prioritize` starts the prioritization process, which will end by
+removing the `I-prioritize` label and appending one of the 4 labels we will discuss below:
+
+- `P-critical`
+- `P-high`
+- `P-medium`
+- `P-low`
+
+Each of these labels defines a strategy the team will adopt regarding:
+
+- The amount of focus a given issue will receive
+- How members of the community can get involved
+
+## P-critical
+A `P-critical` issue is a potentially blocker issue. These issues will be raised at the compiler
+team's triage meeting on a weekly basis.
```

Comment:
I'm not sure about my wording, I just wanted to convey what kind of "blocker" `P-critical` are. Makes sense?

```suggestion
A `P-critical` is an issue potentially blocking a compiler release (i.e. highly recommended to be solved before a new compiler release). These issues will be raised at the compiler
team's triage meeting on a weekly basis.
```

Reply:
Yeah, I think that's more accurate to how `P-critical` is used (not just compiler release, but the general toolchain release too).

Reply:
Fixed